feature.env

```diff
@@ -1,6 +1,4 @@
 PORT=3007
-HMPPS_AUTH_URL=http://localhost:9091/auth
-TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
+HMPPS_AUTH_URL=http://localhost:8090/auth
+TOKEN_VERIFICATION_API_URL=http://localhost:8092/verification
 NODE_ENV=development
-CLIENT_ID=clientid
```

Comment:
i actually don't know what this stuff is really for. removed the env vars to hopefully break something then we'll find out...